gitlab/cli.py

```diff
@@ -241,6 +244,6 @@ def main():
     if debug:
         gl.enable_debug()
 
-    gitlab.v4.cli.run(gl, what, action, args, verbose, output, fields)
+    gitlab.v4.cli.run(gl, what, action, args_dict, verbose, output, fields)
 
-    sys.exit(0)
+    return 0
```

Comment:
is there even a need to return anything here? This is Python, not C. Maybe we can delete the line?

Reply:
Actually we should.

```
$ cat ~/.local/bin/gitlab
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from gitlab.cli import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

So it returns the result code via the sys.exit(main())

Reply:
> is there even a need to return anything here? This is Python, not C. Maybe we can delete the line?

Probably stuff already known: As a note. All programs return an exit code. Exit code value is from 0-255 (I believe). Unix considers an exit code of 0 to be success and any other result a failure.

In Python if you return None, which is the default if a function doesn't return anything, then sys.exit(None) will return an exit code of 0.

Reply:
Agreed we probably don't even need this line. From what I've seen elsewhere the `main` of the CLI module usually explicitly exits rather than returns (something like `sys.exit(return_code)`) but in our case the `die()` function already does a `sys.exit(1)` if there are any exceptions from the CLI so if it gets to this point it's definitely zero.

Reply:
I have removed the `return 0`